ext/random/random.c

```diff
@@ -110,7 +106,8 @@ PHPAPI uint32_t php_random_range32(php_random_algo_with_state engine, uint32_t u
 	}
 
 	/* Ceiling under which UINT32_MAX % max == 0 */
-	limit = UINT32_MAX - (UINT32_MAX % umax) - 1;
+	uint32_t limit = UINT32_MAX - (UINT32_MAX % umax) - 1;
+	uint32_t count = 0;
 
 	/* Discard numbers over the limit to avoid modulo bias */
 	while (UNEXPECTED(result > limit)) {
```

Comment:
```suggestion

	/* Discard numbers over the limit to avoid modulo bias */
	uint32_t count = 0;
	while (UNEXPECTED(result > limit)) {
```
This grouping is a little more logical to me.